modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -60,97 +36,91 @@ output application/dw
-
+_Experimental:_ Dumps the input context and the failing script into a folder when this property is set to true. This behavior enables you to track the failing script and the data that makes the script fail, which is particularly useful for verifying that the received input data is valid. Incorrect scripts often fail when an upstream component generates invalid data.
 * Type: `Boolean`
 * Default: `false`
-
 `com.mulesoft.dw.dump_files` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave. See xref:dataweave-troubleshoot.adoc[Troubleshooting a Failing DataWeave Script].
-
-
 //com.mulesoft.dw.dump_folder//
 |`com.mulesoft.dw.dump_folder`|
 _Experimental:_ Specifies the path in which to dump files when `com.mulesoft.dw.dump_files` is set to `true`.
-
 * Type: `String`
-* Default: `java.io.tmpdir` (system property value)
-
+* Default: `/var/folders/8f/5jqs1kh97k16dzsmxxph2m5c0000gp/T`
 `com.mulesoft.dw.dump_folder` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave. See xref:dataweave-troubleshoot.adoc[Troubleshooting a Failing DataWeave Script].
-
-
 //com.mulesoft.dw.dumper_fill_stacktrace//
 |`com.mulesoft.dw.dumper_fill_stacktrace`|
 _Experimental:_ Adds the StackTrace for dumper exceptions when `com.mulesoft.dw.dump_files` is set to `true`.
-
 * Type: `Boolean`
 * Default: `false`
-
-`com.mulesoft.dw.dumper_fill_stacktrace` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave.
-
-
-//com.mulesoft.dw.error_value_length//
-|`com.mulesoft.dw.error_value_length`|
-Sets the maximum length of exception messages to display to the user. The message is truncated to the maximum length. This setting is useful for avoiding long exception messages.
-
-* Type: `Number`
-* Default: `80`
-
-
+`com.mulesoft.dw.dumper_fill_stacktrace` is an _experimental feature_ that is subject to change or removal from future versions of DataWeave. See xref:dataweave-troubleshoot.adoc[Troubleshooting a Failing DataWeave Script].
 //com.mulesoft.dw.indexsize//
 |`com.mulesoft.dw.indexsize`|
 Sets the maximum size (in bytes) of the page in memory that indexed readers use.
-
-* Type: `Number`
-* Default: `1572864` (1.5 MB)
-
-
+* Type: `Integer`
+* Default: `1572864`
+//com.mulesoft.dw.java.stacktrace//
+|`com.mulesoft.dw.java.stacktrace`|
+Specifies the depth of the java stacktrace
+* Type: `Integer`
+* Default: `6`
 //com.mulesoft.dw.max_memory_allocation//
 |`com.mulesoft.dw.max_memory_allocation`|
-Sets the size (in bytes) of each slot in the off-heap memory pool.
-
-* Type: `Number` (in bytes)
-* Default: `1572864` (1.5 MB)
-
-See xref:dataweave-memory-management.adoc[DataWeave Memory Management].
```

Comment:
need to update in the source code and add this to 2.5.

Reply:
manually updated PR, though since we are saying (in bytes), perhaps we don't need the 1.5 MB beside the default value that's in bytes

Reply:
added to source